### ISSUE TITLE
Ratchet and Clank. Options appear for FG, counters report increase, but game does not reflect FG.

### ISSUE THREAD

**Issue by TimmyP777** [Nukem9/dlssg-to-fsr3#171]
It "ungreys" the option in the game, but doesn't work.

Havent seen any reports of this game.

**Comment by frozenking95** [comment]
yep game feels little choppy

**Comment by REB-ro** [comment]
It works. I'm testing it right now in this game. But there are nasty glitches in the captions and on-screen stat information (static images like HP and ammo). Anything else looks pretty impressive, works like normal DLSS 3.0

**Comment by merlinch** [comment]
what? i've been playing for the past 6 hours with this mod. Since i using 1660ti i needed the nvapi mod, and using either fsr 2.2 or xess (dlss doesnt work, duh, gtx card)[https://cdn.discordapp.com/attachments/989635740545740800/1187293762049822730/2023-12-20_16-35-081.mp4?ex=65965c7b&is=6583e77b&hm=20fa97f91444c9ad25ea26065204b31b52c9eb5974083de23fa0ff8da512c498&](url)

**Comment by cichy45** [comment]
@merlinch can you write something more about the nvapi mod? What is it, where can you get it?

**Comment by TimmyP777** [comment]
> @merlinch can you write something more about the nvapi mod? What is it, where can you get it?

Hes not using this mod, he is using LukeFZ mod. This mod, for using DLSS upscaling with FSR3 Frame generation is most definitely not working. The user above does not know what he or she is talking about.

**Comment by merlinch** [comment]
> > @merlinch can you write something more about the nvapi mod? What is it, where can you get it?
> 
> Hes not using this mod, he is using LukeFZ mod. This mod, for using DLSS upscaling with FSR3 Frame generation is most definitely not working. The user above does not know what he or she is talking about.

![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/45236574/5281420e-a91d-4229-974e-02ebd81c7b1c)
You sure about that? I use FakeMichau NVAPI Dummy from https://github.com/FakeMichau/nvapi-dummy

No it's not LukeFZ mod

**Comment by TimmyP777** [comment]
Well perhaps you could be clearer next time?

Anyways this doesn't seem to work with DLSS. It doesn't render correctly; all black with some effects it actually kinda strange. With FSR it works, but it still ultimately runs better with DLSS2.

Hope the dev can take a look at this.

**Comment by davipinhati** [comment]
tbh this is one of the games i felt a huge difference, i got a increase of 80 fps in hatchet (native 70, fsr3 150), but i got some hud artifacts.

**Comment by TimmyP777** [comment]
There has to be a problems somewhere then. Some sort of compatibility issue. This works great in everything, but does nothing in this despite even appearing to work in fps counter. It runs worse with it on.

**Comment by TimmyP777** [comment]
Bump this once.

Ive tried everything. In every single scenario FG lowers performance. It will increase the fps counter, but play choppier. If I disable and just use DLSS its back to smooth. Funny thing is that despite playing much smoother, after doing this the fps counter reports half fps.

**Comment by TimmyP777** [comment]
Its RT. 

If you have it enabled, using the mod lower performance. About -10fps.

If you have RT disabled, you get the nice increase 40-60fps.

**Comment by REB-ro** [comment]
Guys, this mod works if you activate HAGS. I also tested it with DLSS2 quality mode vrs 2.5.1 (best in my opinion).

Works perfectly, apart from the UI/HUD bug. I also found out that the higher the FPS counter, the more HUD artifacts. Playing at maximum graphics possible to get around 60-70 fps (with FSR 3) reduces HUD artifacts a bit (still ugly, but less artifacts). For some reason, I didn't experience any input lag either (wireless controller!!!). This mod is really good.

Also, disable any RBar feature in this game. For some reason it causes problems and huge FPS drop in-game. Enjoy.

**Comment by TimmyP777** [comment]
Does RT affect your performance negatively? I literally have 50% increases in every game, this is the only one giving problems. Not even Puredark mods are giving issues.

**Comment by cichy45** [comment]
> Does RT affect your performance negatively?

No. I mean, if I enable RT then performance goes down because of bigger GPU load. But enabling FG will increase FPS as expected ~60%.

**Comment by TimmyP777** [comment]
I can run everything now at the highest settings on my 3070 at 1440p DLSSb, except this one game. Oh well, its still 50-60 with rt anyways so no big deal. To be clear, with RT on enabling this lowers my performance.

**Comment by kedaitinh12** [comment]
For LukeFZ 0.7.2 you can use here
https://drive.google.com/file/d/1-vgQPeIk_7wpgjiqqiqmoQUlrvDi0X6F/view?usp=drivesdk

And read here to know what file to use. Read redme in file download to know how to mod
https://docs.google.com/spreadsheets/d/1bGDjgkRahwNvD2XtqiHdZBRc6VB52UKsr1Qn1TmvSHo/edit#gid=0

**Comment by TimmyP777** [comment]
LukeFZ mods dont use DLSS2 for upscaling and therefore are completely different from this mod, which is only for Nvidia 2000/3000.

**Comment by kedaitinh12** [comment]
> LukeFZ mods dont use DLSS2 for upscaling and therefore are completely different from this mod, which is only for Nvidia 2000/3000.

Nukem9 mod only use for RTX 2000/3000 when LukeFZ mod can use for GTX and AMD card, read second link i shared above

**Comment by TimmyP777** [comment]
> > LukeFZ mods dont use DLSS2 for upscaling and therefore are completely different from this mod, which is only for Nvidia 2000/3000.
> 
> Nukem9 mod only use for RTX 2000/3000 when LukeFZ mod can use for GTX and AMD card, read second link i shared above

This mod is explicitly for RTX cards that feature DLSS upscaling\reconstruction but not frame generation. It has no use for any other cards, and being that LukeFZ mods do not have this functionality, they are not relevant here in any way.

**Comment by kedaitinh12** [comment]
> > > LukeFZ mods dont use DLSS2 for upscaling and therefore are completely different from this mod, which is only for Nvidia 2000/3000.
> > 
> > 
> > Nukem9 mod only use for RTX 2000/3000 when LukeFZ mod can use for GTX and AMD card, read second link i shared above
> 
> This mod is explicitly for RTX cards that feature DLSS upscaling\reconstruction but not frame generation. It has no use for any other cards, and being that LukeFZ mods do not have this functionality, they are not relevant here in any way.
> 
> Read what I said above, then read about the place you are currently at. THANKS!

LukeFZ can use DLSS frame generation for some game, some game use FSR2. I did some test from some game

**Comment by TimmyP777** [comment]
> > > > LukeFZ mods dont use DLSS2 for upscaling and therefore are completely different from this mod, which is only for Nvidia 2000/3000.
> > > 
> > > 
> > > Nukem9 mod only use for RTX 2000/3000 when LukeFZ mod can use for GTX and AMD card, read second link i shared above
> > 
> > 
> > This mod is explicitly for RTX cards that feature DLSS upscaling\reconstruction but not frame generation. It has no use for any other cards, and being that LukeFZ mods do not have this functionality, they are not relevant here in any way.
> > Read what I said above, then read about the place you are currently at. THANKS!
> 
> LukeFZ can use DLSS frame generation for some game, some game use FSR2. I did some test from some game

Again. This mod is EXPLICITLY for combining DLSS2 and FSR3 frame generation, to essentially enable DLSS3 Frame Generation on NVIDIA RTX cards that are not series 4000, because they already contain DLSS frame generation support.

This is the only solution available so far, for this singlular purpose.
